packages/pretty-format/src/index.js

```diff
@@ -790,6 +790,7 @@ const DEFAULTS: Options = {
   printFunctionName: true,
   spacing: '\n',
   theme: {
+    comment: 'gray',
```

Comment:
No idea if this is considered a breaking change, but the next version is a major anyways